app/controllers/concerns/sufia/file_sets_controller_behavior.rb

```diff
@@ -5,7 +5,7 @@ module FilesController
     autoload :LocalIngestBehavior
```

Comment:
@awead I see that on line 2 we're still referencing a `FilesController` and we have some directory structure around this to pull in the other modules. Do you think this PR should expand to nuke any and all references to "FilesController" (or "files_controller") in the code, the filenames, and directory names? Or shall we merge this PR and ticket that as a follow-on?